client/components/findStar.jsx

```diff
@@ -29,33 +29,90 @@ class FindStar extends Component {
             Celebrity: [],
             click: '',
             isLoaded : false,
-            imgPath : ''
+            imgPath : '',
+            isFavorite: false
         };
         this.handleClick = this.handleClick.bind(this);
-        this.handleFavorite = this.handleFavorite.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
     }
     
     componentDidMount(){
         fetch('http://localhost:5000/list')
         .then((res) => res.json())
         .then((data) => {
             this.setState({Celebrity:data}, () => console.log(this.state.Celebrity));
-            console.log(this.props);
         }
     )}
 
     handleClick(event){
-        this.setState({click: event.target.id, isLoaded: true}, () => console.log(this.state.click));
+        this.setState({click: event.target.id, isLoaded: true}, () => {
+            console.log(this.state.click);
+
+            //Check if selected celebrity is in the favorite list
+            var data = {username: 'test1', favorite: this.state.click};
+            fetch('http://localhost:5000/favButton', {
```

Comment:
please use more descriptive names for functions/variables you create.
In many cases, front/backend dev work independently and names should tell what it does.
favButton does not tell me anything as a server side dev.
use something like `getFavoriteState` or `isFavorite`.